docs/source/model_design/vivarium_model_components/GBD_disease_health/index.rst

```diff
@@ -30,14 +30,137 @@ Understanding GBD's Disease and Health Categories
 .. contents::
   :local:
 
-GBD Definitions
----------------
+This document seeks to clarify a few different disease and health concepts in 
+GBD. These are: a cause, a sequela, a health state and an impairment. First, we 
+will explain these different concepts conceptual. The official GBD defintions 
+can be found at the bottom of the page. 
 
-- Include the GBD definitions for each of these different categories 
+Conceptual Definitions within the GBD Hierarchy Context
+-------------------------------------------------------
 
-Understanding the GBD Hierarchy
--------------------------------
+To understand these overlapping defintions, it is helpful to think of them 
+in the context of a specific example. We will use cardiovascular disease for this. 
+In particular, we will examine: 
 
-- Showing what is a direct subset, what overlaps, what interacts but is independent 
-- Include some information on modeling implications as a result of this 
+- Causes: ischemic heart disease (IHD) and hypertensive heart disease (HHD) 
+- Health states: medically managed, mild, moderate and severe heart failure 
+- Impairments: heart failure 
+- Sequelae: numerous, including "severe heart failure due to IHD" and "AMI first 2 days" among others 
 
+To help with understanding some of the below information, a diagram of the heart failure impairment is provided: 
+
+.. image:: health_state_impairment_sequela.svg
+
+
+Causes
+++++++
+
+The cause hierarchy diagram for IHD and HHD is included here as a reference. 
+
+.. image:: cause_hierarchy_ihd_hhd.svg
+
+As you can see, the cause hierarchy slowly breaks down cause groups into more specific 
+diseases, moving from all non-communicable diseases to specific types of cardiovascular 
+disease. At each level (labeled), the cause hierarchy is mutually exclusive and 
+collectively exhaustive - meaning all disease and death can be placed into a group at 
+each level of the diagram. A cause is anything that causes death or disability. 
+
+Both IHD and HHD are within the more general cardiovascular disease cause (c_491). IHD and HHD 
+are "causes" within GBD. 
+
+Sequelae
+++++++++
+
+Once you have selected a specific cause - for example, IHD - you can then identify the 
+sequelae within that cause. These are all of the items listed below IHD and HHD in the 
+diagrams above. 
+
+Sequelae detail a specific medical condition associated with 
+a cause. For example, IHD has medical conditions like acute myocardial infarction 
+(a heart attack) and angina (chest pain) as associated sequelae. These are the things 
+a patient might be experiencing, or a doctor might identify. 
+
+Sequelae often contain a level of severity for that condition - for example, mild and moderate 
+angina are both sequelae within IHD. 
+
+Health States
++++++++++++++
+
+A health state is the amount of health or disability within a sequelae. This is summarized 
```

Comment:
Well, technically I don't think a health state is an "amount" -- it seems like the health state is better conceptualized as a "state of being" (not sure how to say it without using the word "state"), which is described in terms of your symptoms, etc., and the _amount_ of health/disability for that state is quantified by the disability weight. But maybe this is getting a bit pedantic, so I'm not sure if it's worth trying to make this distinction.

Reply:
I edited the language a bit to try and capture this